municipal_finance/settings.py

```diff
@@ -450,30 +450,23 @@
     "ack_failures": True,  # Dequeue failed tasks
 }
 
-if not DEBUG:
-    DEFAULT_FILE_STORAGE = "municipal_finance.storage.MediaStorage"
-    AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID", "")
-    AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY", "")
-    AWS_STORAGE_BUCKET_NAME = env.str(
-        "AWS_STORAGE_BUCKET_NAME", "munimoney-media"
-    )
-    AWS_S3_CUSTOM_DOMAIN = "%s.s3.amazonaws.com" % AWS_STORAGE_BUCKET_NAME
-    AWS_S3_OBJECT_PARAMETERS = {
-        "CacheControl": "max-age=86400",
-    }
-else:
-    DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
-    AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID")
-    AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY")
-    AWS_STORAGE_BUCKET_NAME = env.str("AWS_STORAGE_BUCKET_NAME")
-    AWS_DEFAULT_ACL = "public-read"
-    AWS_BUCKET_ACL = "public-read"
-    AWS_AUTO_CREATE_BUCKET = True
-    AWS_S3_ENDPOINT_URL = env.str("AWS_S3_ENDPOINT_URL", None)
-    AWS_S3_REGION_NAME = env.str("AWS_S3_REGION_NAME", None)
-    AWS_S3_SECURE_URLS = env.bool("AWS_S3_SECURE_URLS", True)
-    AWS_S3_CUSTOM_DOMAIN = env.str("AWS_S3_CUSTOM_DOMAIN", None)
-    AWS_S3_FILE_OVERWRITE = False
+DEFAULT_FILE_STORAGE = "municipal_finance.storage.MediaStorage"
+AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID", "")
+AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY", "")
+AWS_STORAGE_BUCKET_NAME = env.str(
+    "AWS_STORAGE_BUCKET_NAME", "munimoney-media"
+)
+AWS_DEFAULT_ACL = "public-read"
+AWS_BUCKET_ACL = "public-read"
+AWS_AUTO_CREATE_BUCKET = True
+AWS_S3_ENDPOINT_URL = env.str("AWS_S3_ENDPOINT_URL", None)
+AWS_S3_REGION_NAME = env.str("AWS_S3_REGION_NAME", None)
+AWS_S3_SECURE_URLS = env.bool("AWS_S3_SECURE_URLS", True)
+AWS_S3_CUSTOM_DOMAIN = "%s.s3.amazonaws.com" % AWS_STORAGE_BUCKET_NAME
+AWS_S3_FILE_OVERWRITE = False
```

Comment:
we didn't used to have this in prod (DEBUG=False) so I don't think we should add it now